src/libasr/ASR.asdl

```diff
@@ -339,7 +339,7 @@ expr
 --     and Fortran: in Python "Booleans in Python are implemented as a subclass
 --     of integers", in Fortran the "default logical kind has the same storage
 --     size as the default integer"; we currently use kind=4 as default
---     integer, so we also use kind=4 for the default logical.)
+--     integer, so we also use kind=4 for the default logic.)
```

Comment:
```suggestion
--     integer, so we also use kind=4 for the default logical.)
```

Here logical refers to logical type and not a logic.

Reply:
why not "... for the default logical type" ?